Xabe.FFmpeg/Conversion/IConversion.cs

```diff
@@ -37,6 +37,20 @@ public interface IConversion
         /// <returns></returns>
         IConversion SetPriority(ProcessPriorityClass? priority);
 
+        /// <summary>
+        /// Extracts Every frameNo frame of the input video and outputs as a png image
+        /// </summary>
+        /// <param name="frameNo">The frame interval to extract </param>
+        /// <returns></returns>
+        IConversion ExtractEveryNthFrame(int frameNo);

```

Comment:
This function has to get one additional parameter. 
Users don't know that they have to name file with _%d
Func<string, string> where the pass in argument will be the number, and it return's name with that number. User could define where in the output name will be the number.

In the Conversion class you have to add default function which returns output file name and override it in one of ExtractNthFrame functions.

If you will have problems with that, or it's to complited just tell me. I could help :)

Reply:
Hi

I should be able to work it out. I’ll have a look tomorrow evening after work.

Thanks for the comments

Reply:
Hi

I should be able to work it out. I’ll have a look tomorrow evening after work.

Thanks for the comments